### ISSUE TITLE
Setting to disable container-updates on specific hosts

### ISSUE THREAD

**Issue by maxulm** [topgrade-rs/topgrade#967]
## I want to suggest a new step

* Which tool is this about? Where is its repository?
* Which operating systems are supported by this tool?
* What should Topgrade do to figure out if the tool needs to be invoked?
* Which exact commands should Topgrade run?
* Does it have a `--dry-run` option? i.e., print what should be done and exit
* Does it need the user to confirm the execution? And does it provide a `--yes`
  option to skip this step?

## I want to suggest some general feature
Topgrade should be able to disable container-updates on a specific host

**Comment by SteveLauC** [comment]
I am not quite sure I understand this feature, would you like to elaborate a bit? What's Topgrade's current behavior?

**Comment by maxulm** [comment]
I have several servers automatically updated with topgrade.
On some of them, I don't want docker-containers to be updated automatically on others it should be done.
It's a bit difficult to explain. Hope you got it.

**Comment by SteveLauC** [comment]
Ok, I get it. Currently, you have to configure your servers separately, on the servers where you don't want Docker/Podman images to be updated, disable the `containers` step.

**Comment by maxulm** [comment]
So topgrade has a local config on the "slaves" and not just the config on the "master"?

**Comment by SteveLauC** [comment]
> So topgrade has a local config on the "slaves" and not just the config on the "master"?

True, every node should have its own configuration file or the default configuration will be used. The only thing that Topgrade's remote functionality will do is launch a Topgrade process on the remote node. And, Topgrade does not have a concept of cluster, they are all equivalent nodes.

**Comment by maxulm** [comment]
Great! Thank you that much :)
Von meinem iPhone gesendet

Am 24.10.2024 um 07:49 schrieb SteveLauC ***@***.***>:

﻿

So topgrade has a local config on the "slaves" and not just the config on the "master"?

True, every node should have its own configuration file or the default configuration will be used. The only thing that Topgrade's remote functionality will do is launch a Topgrade process on the remote node. And, Topgrade does not have a concept of cluster, they are all equivalent nodes.

**Comment by GideonBear** [comment]
Closing because this was resolved
